style.css

```diff
@@ -222,9 +222,12 @@ a {
   padding-left: 10ps;
 }
 
-  grid-template: 
-    "profile-pic-bottom username three-dots-icon" a /
+.profile-info-grid {
+    display: grid;
+  grid-template : 
+    "profile-pic-bottom username three-dots-icon" auto /
     auto auto auto;
```

Comment:
Grid doesn't work.  Was added late at the end, needed more time to complete.  Should be inline in column not shown as a row.

Reply:
Borders could have been added to sections to distinguish between each other.